elasticdl/python/common/k8s_client.py

```diff
@@ -353,6 +364,16 @@ def create_tensorboard_service(
             owner=self.get_master_pod(),
         )
 
+    def create_ps_service(self, ps_id):
+        return self._create_service(
+            name=self.get_ps_pod_name(ps_id),
+            port=2222,
+            target_port=2222,
```

Comment:
Should we move all the ports including the ones for tensorboard to a central place so it’s easier to see what ports are being used and can be reused?